src/tests/ngxs.setup.spec.ts

```diff
@@ -1,5 +1,11 @@
-import { Action, NgxsAfterBootstrap, NgxsOnInit, State, StateContext } from '@ngxs/store';
+import { Action, NgxsAfterBootstrap, NgxsModule, NgxsOnInit, Select, Selector, State, StateContext } from '@ngxs/store';
 import { NgxsTestBed } from '../lib/ngxs.setup';
+import { Component } from '@angular/core';
+import { Observable, Subject } from 'rxjs';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { CommonModule } from '@angular/common';
+import { mockSelect } from '../jest/src/public_api';
```

Comment:
maybe `import { mockSelect } from '@ngxs-labs/testing/jest'`

Reply:
This won't work as it is inside the lib (the same goes for the NgxsTestBed import at line 2 `import { NgxsTestBed } from '../lib/ngxs.setup';
`)
Do you want me to add path aliases?

Reply:
> Do you want me to add path aliases?

Yes

Reply:
Done 👍